app-cryptography/signing-party/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=signing-party
+PKGSEC=misc
+PKGDES="Various OpenPGP related tools made for key signing party"
+PKGDEP="gnupg perl-class-methodmaker perl-mailtools perl-mime-tools perl-net-idn-encode perl-term-readkey perl-text-template \
+	perl-gnupg-interface imagemagick python-3 perl-gd qrencode qprint"
+BUILDDEP="perl gnupg"
+# TeXLive: for LaTeX support in gpg-key2latex.
+# Postfix: for mailing the encrypted messages containing signatures
```

Comment:
Ditto.

Reply:
Missing MTA does not trigger an error, a raw text email file will be exported to the CWD.

Reply:
Instead, if postfix is installed but not configured properly, it will report an error. So the best solution here is to just leave it in PKGSUG.